docs/cmd_list.txt

```diff
@@ -39,6 +39,7 @@ Workspace file management commands:
   du        Get workspace disk usage.
   ls        List workspace files.
   mv        Move files within workspace.
+  prune     Prune the workspace.
```

Comment:
"Prune workspace files" may be more in line with the other existing options.

We could also consider using somewhat longer version "Prune intermediate workspace files" but that is perhaps too long; and if we add two options `--include-inputs` and `--include-outputs` (see below) it may be good to keep the help rather generic (i.e. without "intermediate).